client/state/selectors/test/is-site-using-full-site-editing.js

```diff
@@ -0,0 +1,57 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import isSiteUsingFullSiteEditing from '../is-site-using-full-site-editing';
+
+describe( 'isSiteUsingFullSiteEditing', () => {
+	test( 'returns false if site does not exist', () => {
+		const isFSE = isSiteUsingFullSiteEditing( {}, 1 );
```

Comment:
Try passing in `undefined` (not initialized) or mock more of the state structure. The sites reducer should add that prop after the reducer is called.

Reply:
Another option is bulletproofing getRawSites for the case, but I'll leave it up to you!